ntoskrnl/include/internal/io.h

```diff
@@ -321,7 +321,7 @@ typedef struct _IO_WORKITEM
 typedef struct _IO_INTERRUPT
 {
     KINTERRUPT FirstInterrupt;
-    PKINTERRUPT Interrupt[MAXIMUM_PROCESSORS];
+    PKINTERRUPT Interrupt[MAXIMUM_PROCESSORS - 1];
     KSPIN_LOCK SpinLock;
```

Comment:
Is this spinlock mandatory, or could we restore using the `FirstInterrupt`'s one?